port/zos390/omrvmem.c

```diff
@@ -1352,9 +1369,26 @@ omrvmem_find_valid_page_size(struct OMRPortLibrary *portLibrary, uintptr_t mode,
 		uintptr_t defaultLargePageSize = 0;
 		uintptr_t defaultLargePageFlags = OMRPORT_VMEM_PAGE_FLAG_NOT_USED;
 
-		/* If the page type is NOT_USED (as with the -Xlp<size> options)
-		 * the legacy behavior is to default to FIXED pages.
-		 */
+		/* If the page type is PREFER_PAGEABLE try to get 1M pageable large pages. Otherwise, use a fixed large page. */
+		if ((IS_VMEM_PAGE_FLAG_PAGEABLE_PREFERABLE(validPageFlags))) {
+
+			/* Try 1M large pages */ 
+			if (ONE_M == validPageSize) {
+				/* Keep from calling this again if ONE_M fails. */
+				isPageableAttempted = TRUE;
+
+				pageSizeFound = isLargePageSizeSupported(portLibrary, ONE_M, OMRPORT_VMEM_PAGE_FLAG_PAGEABLE);
+				if (TRUE == pageSizeFound) {
```

Comment:
the same here: it can be just `if (pageSizeFound) {`

Reply:
I see that construction `TRUE ==` is used in a few places already, so I guess it is good as is. Ignore this comment